### DISCUSSION TITLE
PorousFlow Permeable BCs

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#20165] [Q&A General] [answered]
Hello,

I want to simulate a porous body which is subjected to a ramp displacement (`FunctionDirichletBC`) at the top. The sides and bottom are confined, the top is unconfined so fluid flow should occur there, but the `FunctionDirichletBC` makes it impermeable. How can I make my BC permeable for the fluid?

**Comment by WilkAndy** [comment, marked as answer]
Short answer: see https://mooseframework.inl.gov/modules/porous_flow/boundaries.html

Longer answer: this is difficult both conceptually and numerically/physically.  That is, you are coupling the porous flow within your sample to some other physics that governs the fluid's physics in the atmosphere.  The rate of fluid transfer between the two obviously depends on the physics in the atmosphere.  So what is that physics?   You have to decide, and the above link gives you some suggestions.  For the type of problem that i'm often doing recently, i make a thin,  permeable layer and then use a `PorousFlowOutflow` BC, but in the past i've used the `PorousFlowSink` to good effect

**Reply by Flolaffel** [reply]
Thank, it works now with `PorousFlowOutflow`. I tried that before and it didn't work. Guess I just got some settings wrong the first time.
